docs/DeveloperGuide.md

```diff
@@ -342,17 +382,60 @@ Priorities: High (must have) - `HIGH`, Medium (nice to have) - `MEDIUM`, Low (un
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified
-otherwise)
+(For all use cases below, the **System** is the `PennyWise` and the **Actor** is the `User`, unless specified
+otherwise, for all **lists**, the `User` is able to select between the `expenditure` or `income` lists.)
+
+**Use case: UC1 - Add an entry**
```

Comment:
should we say income/expenditure instead of entry?

Reply:
For this I kept it as entry because all the Use Cases were generalised as they are either expenditure and income, I have specified this at the start of the Use Case section, such that all entries only refer to `expenditure` or `income` only.